ntbs-service/Migrations/20191003093900_AddSampleTbServiceAdGroups.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class AddSampleTbServiceAdGroups : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            //                          table, key column,   key value, change column, change value
```

Comment:
Are we adding the ad codes for all the other services at a later time? Maybe add a TODO if so?

Reply:
Yep, for two reasons:
- we only have a handful of groups set up on the the SW AD which we'll use for testing
- we don't have the full list available at this point anyways!

A TODO inside the migration doesn't seem to make sense to me - editing a migration that's been run is not something EF supports afaict. If it did, I'd be well up for that (similar to [liquibase's `runOnChange` option](https://www.liquibase.org/documentation/changeset.html#available-attributes)!

If you have another idea for where to put the TODO then keen for that!

Reply:
Yeah I agree a TODO in a migration is a bit strange... I think it's fine to just add at a later date in that case, as we're just adding more data